README.md

```diff
@@ -86,6 +88,35 @@ $ npm config set script-shell "C:\\Program Files\\Git\\bin\\bash.exe"
 
 You must have [Git for Windows](https://git-scm.com/download/win) installed. See this [StackOverflow answer](https://stackoverflow.com/a/46006249) for more details.
 
+### Build the bot database
+
+
+*As we use prisma for managing our DB it is up to you what relational database framework you use.*
+
+[The following is a guide to setting up Postgres inside a docker container.](https://github.com/docker-library/docs/blob/master/postgres/README.md) While the choice of database is up to you, the instructions for getting started in this guide assume you are using a postgres docker image. 
+After you have Postgres (or your DB of choice) up and running edit the line in your .env file to be 
```

Comment:
Is this newline meant to start a separate paragraph? GitHub renders this as one paragraph:

![Screen Shot 2022-10-09 at 10 19 44](https://user-images.githubusercontent.com/1500092/194767900-538ddd90-a834-4544-9044-7983c477f4df.png)

```suggestion

After you have Postgres (or your DB of choice) up and running edit the line in your .env file to be 
```